feature-libs/checkout/components/components/delivery-mode/delivery-mode.component.html

```diff
@@ -1,128 +1,65 @@
-<ng-container *cxFeatureLevel="'!4.2.2'">
```

Comment:
Since we no longer use the `cxFeatureLevel`, can you remove the module import (`FeaturesConfigModule`) from `DeliveryModeModule`?